src/core/ddsc/tests/deadline.c

```diff
@@ -30,6 +30,7 @@
 
```

Comment:
It is pretty trivial to split this commit into one that fixes the deadline test and one that makes the functional changes on setting the listeners. I think doing so would be worth it.

Reply:
I guess that means we will do the deadline fix in a separate pull request?